BridgeSDK/BridgeAPI/Categories/UIDevice+Hardware.h

```diff
@@ -50,11 +53,16 @@
 #define IPAD_MINI3_NAMESTRING           @"iPad Mini 3"
 #define IPAD_MINI4_NAMESTRING           @"iPad Mini 4"
 #define IPAD_AIR2_NAMESTRING            @"iPad Air 2"
-#define IPAD_PRO_12_9_1G_NAMESTRING     @"iPad Pro (12.9\")"
-#define IPAD_PRO_9_7_NAMESTRING         @"iPad Pro (9.7\")"
+#define IPAD_PRO_12_9_1G_NAMESTRING     @"iPad Pro, 12.9\""
```

Comment:
Why the extra quotation mark?

Reply:
it's an inch mark (12.9 inch iPad Pro--that's how Apple distinguishes them)